discovery-deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: discovery-agent
+  labels:
+    name: discovery-agent
+    instance: discovery
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: discovery-agent
+      app.kubernetes.io/instance: discovery
+  template:
+    metadata:
+      annotations:
+      labels:
+        app.kubernetes.io/name: discovery-agent
+        app.kubernetes.io/instance: discovery
+    spec:
+#      imagePullSecrets:
+#        - name: <Secret_Name>
+      containers:
+        - env:
+            - name: CENTRAL_AUTH_CLIENTID
+              value:
+            - name: CENTRAL_AUTH_PRIVATEKEY
+              value: /keys/privateKey
+            - name: CENTRAL_AUTH_PUBLICKEY
+              value: /keys/publicKey
+            - name: CENTRAL_POLLINTERVAL
+              value: 20s
+            - name: CENTRAL_ENVIRONMENT
+              value:
+            - name: CENTRAL_TEAM
+              value:
+            - name: CENTRAL_ORGANIZATIONID
```

Comment:
Do we need to export empty vars?

Reply:
The user using the manifest files will populate these fields.